components/molecule/badgeCounter/src/index.scss

```diff
@@ -2,12 +2,16 @@
 
 $bg-molecule-badge-counter: #ff0057 !default;
 $c-molecule-badge-counter: $c-white !default;
-$br-molecule-badge-counter: $bdw-xl !default;
+$br-molecule-badge-counter: 16px !default;
```

Comment:
$bdrs-molecule-badge-counter: calc(2 * $bdrs-xl) !default;
(sui-theme)[https://github.com/SUI-Components/sui-theme/blob/master/src/settings/_box-style.scss]

Reply:
we can leave it as @andresin87 suggests here, so we don't "mareamos más la perdiz", but this adjustment is assuming all themes have a defined value for $bdrs-xl of 8px (which is not correct)

Reply:
This can become a major version upgrade. Do u agree?

Reply:
yeapi, I agree we can put the `calc(2 * $bdrs-xl) !default;` in a major, after checking the side effects

Reply:
We can do a major for `$bg` too, because it must be `$bgc`. But I think it's too much.

Reply:
`br-molecule-badge-counter`--> `bdrs-molecule-badge-counter`is the major i think. cuz it refers to border-radius

Reply:
> we can leave it as @andresin87 suggests here, so we don't "mareamos más la perdiz", but this adjustment is assuming all themes have a defined value for $bdrs-xl of 8px (which is not correct)

maybe `16px`is ok, but the token naming is not